### ISSUE TITLE
how is the segmentation performance on yolo dataset?

### ISSUE THREAD

**Issue by hihiok** [ArgoHA/D-FINE-seg#62]

**Comment by ArgoHA** [comment]
There is no particular "YOLO dataset", YOLO is a format or a model family. All the metrics I've collected are in the readme

**Comment by hihiok** [comment]
> There is no particular "YOLO dataset", YOLO is a format or a model family. All the metrics I've collected are in the readme

hi sorry for the typo. i meant coco dataset. i would like to know if you did segmentation experiments on coco dataset?

**Comment by ArgoHA** [comment]
I finetuned detection coco weights on the COCO segmentation dataset and I have metrics like F1 score, IoU. I can't compute mask mAPs on my gpu because of vRAM limitation (when I fine tune - I don't store all mask predictions as they don't fit). To compute metrics for TACO dataset I rent a machine with a ton of vRAM

**Comment by hihiok** [comment]
> I finetuned detection coco weights on the COCO segmentation dataset and I have metrics like F1 score, IoU. I can't compute mask mAPs on my gpu because of vRAM limitation (when I fine tune - I don't store all mask predictions as they don't fit). To compute metrics for TACO dataset I rent a machine with a ton of vRAM

May you share the performance of nano version model on coco dataset? this is the performance of your dfine_seg_n_coco.pt. it seems poorer than yolo26n. i tested using your bench.py. let me know if anything wrong at my side. thanks!

+---------+-------+-----------+--------+-------+---------+---------+---------+--------+-----------+-------------+----------------+---------+
|         |  f1   | precision | recall |  iou  |   TPs   |   FPs   |   FNs   | mAP_50 | mAP_50_95 | mAP_50_mask | mAP_50_95_mask | latency |
+---------+-------+-----------+--------+-------+---------+---------+---------+--------+-----------+-------------+----------------+---------+
| PyTorch | 0.568 |   0.646   | 0.507  | 0.303 | 18421.0 | 10075.0 | 17914.0 | 0.484  |   0.355   |    0.454    |     0.264      |  91.0   |
+---------+-------+-----------+--------+-------+---------+---------+---------+--------+-----------+-------------+----------------+---------+

**Comment by ArgoHA** [comment]
to get correct mAPs you need to set confidence threshold to 0.001, this one is important. Ideally you should also test both models in the same place
